cmd/dockerd/daemon.go

```diff
@@ -653,6 +655,10 @@ func (cli *DaemonCli) getContainerdDaemonOpts() ([]supervisor.DaemonOpt, error)
 		opts = append(opts, supervisor.WithLogLevel(cli.LogLevel))
 	}
 
+	if logFormat := cli.Config.LogFormat; logFormat != "" && logFormat != log.TextFormat {
```

Comment:
Does it harm anything to just unconditionally/explicitly pass `withLogFormat(log.TextFormat)` in the case of `text`?

Reply:
It doesn't harm anything - being a new contributor to this codebase I just wanted to minimize change to parts of the system as a result of this feature. The only user visible result of passing it through unconditionally will be having an extra debug/format setting in the config file.